main/templates/main/index.html

```diff
@@ -0,0 +1,430 @@
+                    When a user is creating a joind meeting adding other users each time <br> 
+                    can be tidies and annoying when it is repeted multiple times <br>
+                    with the same group of people. <br>
+                    Our soultion is creating a group that the user will define ones and can be changed over time to remove the discomfort of searching <br>
+                    the right users for the meeting each time.
+                </p>
+              </div>
+              <div class="column bgimg-2" id="groups">
+                  <div class="caption">
+                    <span class="border" id = "wallpaperCaption" style="font-size: 5em; color: dimgrey;">Set Groups</span>
+                  </div>
+              </div>
+            </div>
+        </div>
+        
+<!--        <hr>-->
+        <div class="leftBoxText">
+            <div class="row" style="color: white">
+              <div class="column columnBox">
+                  <h2 style="font-size: 4em;"><u>chat</u></h2>
+                <p style="font-size: 3em;">
+                    Users will be able to comunicate with each other using our site within groups and within joined meeting. 
+                </p>
+              </div>
+              <div class="column bgimg-2" id="chat">
+                  <div class="caption">
+                    <span class="border" id = "wallpaperCaption" style="font-size: 5em; color: black;">chat</span>
+                  </div>
+              </div>
+            </div>
+        </div>
+        
+<!--        <hr>-->
+        <div class="leftBoxText">
+            <div class="row" style="color: white">
+              <div class="column columnBox">
+                  <h2 style="font-size: 4em;"><u>External Calendar</u></h2>
+                <p style="font-size: 280%;">
+                    Users will be able to bring there own calander provider and import events <br>
+                    from it to our site to unify the calendars into one. <br>
+                    At this point of time we will be supporting google calendar.
+                </p>
+              </div>
+              <div class="column bgimg-2" id="calndars">
+                  <div class="caption">
+                    <span class="border" id = "wallpaperCaption" style="font-size: 5em; color: black;">External Calendar</span>
+                  </div>
+              </div>
+            </div>
+        </div>
+		
+        <div class="bgimg-4" id="meetTeam">
+		  <div class="caption">
+			<span class="border" id="wallpaperCaption" style="color: white;">Meet The Team:</span>
+		  </div>
+		</div>
+        
+        <div style="background-image: url(https://images.unsplash.com/photo-1534313240698-2384ede3b065?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=1738&q=80); background-size: cover">
+            <div class="teamWrapper">
+                <h3 style="color: white;">Ron</h3>
```

Comment:
maybe add to all names bold to make it more readable

Reply:
all teams names has been modified to bold type, done.